src/data/schedules.json

```diff
@@ -158,7 +159,7 @@
     {
       "name": "No School",
       "isSpecial": true,
-      "dates": ["8/1/2024-8/11/2024", "9/2/2024", "10/3/2024", "10/14/2024", "11/5/2024", "11/27/2024-11/29/2024"],
+      "dates": ["8/1/2024-8/11/2024", "9/2/2024", "10/3/2024", "10/14/2024", "11/5/2024", "11/27/2024-11/29/2024", "12/23/2024-01/06/2025","1/20/2025","2/17/2025","2/28/2025","3/3/2025","3/24/2025-3/28/2025","4/18/2025","5/26/2025","6/19/2025"],
```

Comment:
5/26 and 6/19 don't seem to be designated as days off -- considering we don't have school in June!

Reply:
Ahhh -- those are summer school days/emergency days. We can add them, but they won't really make a huge difference (we don't operate the site during summer schools anyways)